compiler/lib/js_traverse.mli

```diff
@@ -171,7 +171,8 @@ type scope =
 class rename_variable : esm:bool -> object ('a)
   inherit mapper
 
-  method update_state : scope -> Javascript.ident list -> Javascript.statement_list -> 'a
+  method update_state :
+    ?toplevel:bool -> scope -> Javascript.ident list -> Javascript.statement_list -> 'a
```

Comment:
I think we could merge toplevel with the scope data type. We already have `Module` which is somewhat similar.